soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3084,6 +3084,98 @@ void Randomizer::CreateCustomMessages() {
 			"You found a %rGanon's Castle &%wSmall Key!",
 			"Du erhältst einen %rKleinen&Schlüssel%w für %rGanons Schloß%w!",
 			"Vous obtenez une %rPetite Clé %w&du %rChâteau de Ganon%w!"),
+        GIMESSAGE(RG_GUARD_HOUSE_KEY, ITEM_KEY_SMALL,
+			"You found the key to the&%gGuard House%w!",
+            "Du erhältst einen %rKleinen&Schlüssel%w für das %gWachhaus%w!",
```

Comment:
`%gWachhaus%w` &#8594; `%gHaus der Wachen%w`

Reply:
`%rKleinen&Schlüssel%w` &#8594; `%rkleinen&Schlüssel%w`

This one appears multiple times.